### PR TITLE
full example of app using taskKillGracePeriodSeconds

### PR DESCRIPTION
**Pull request by tgermain**
The doc is a bit misleading about  `taskKillGracePeriodSeconds` being a label you add to the `labels` of your application or a primary key of the app definition json. 

Hope it help a bit #4338

### CODE REVIEW

File: docs/docs/health-checks.md [hunk 1]
```diff
@@ -211,6 +211,22 @@ done
 
 To set the necessary grace period, add the `taskKillGracePeriodSeconds` label to your application definition:
```
Comment (jdef) [inline docs/docs/health-checks.md]: s/label/field/ please

**Comment by jdef** [comment]
ok to test

**Comment by tgermain** [comment]
Done, indeed field avoid confusion around yet another label.

File: docs/docs/health-checks.md [hunk 2]
```diff
@@ -209,8 +209,24 @@ while true; do
...
 
+```json
+{
+  "id": "/foo",
+  "instances": 2,
+  "cmd": "sleep 1000",
+  "cpus": 0.1,
+  "disk": 0,
+  "mem": 16,
+  "args": [
```
Comment (gkleiman) [inline docs/docs/health-checks.md]: An app specifying both `cmd` and args` is not valid:

> As of version 0.7.0, Marathon supports an args field in the app JSON. It is invalid to supply both cmd and args for the same app. The behavior of cmd is as in previous releases (the value is wrapped by Mesos via /bin/sh -c '${app.cmd}).

**Pull request merged**